### PR TITLE
[TASK] Replace `git@github.com:` by `git://github.com/`

### PR DESCRIPTION
**Pull request by alexander-nitsche**
See https://github.com/TYPO3-Documentation/T3DocTeam/issues/162 for details.

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -62,14 +62,14 @@ RABBITMQ_MANAGEMENT="http://intercept.ddev.site:15672/"
...
 
 # Where the main core repo on github is located.
-CORE_SPLIT_MAIN_REPO="git@github.com:typo3/typo3.git"
+CORE_SPLIT_MAIN_REPO="git://github.com/typo3/typo3.git"
 # Base path to where the git-split single extension repos reside
-CORE_SPLIT_SINGLE_REPOS_BASE="git@github.com:TYPO3.CMS/"
+CORE_SPLIT_SINGLE_REPOS_BASE="git://github.com/TYPO3.CMS/"
 # Where the main core repo on github is located for version 8.
-CORE_SPLIT_MAIN_REPO_V8="git@github.com:TYPO3GmbH/elts-8.7-release.git"
+CORE_SPLIT_MAIN_REPO_V8="git://github.com/TYPO3GmbH/elts-8.7-release.git"
```
Comment (andreasfernandez) [inline .env:69]: Are you sure about this? If I try to clone with `git clone git://github.com/TYPO3GmbH/elts-8.7-release.git`, I get the following output:
```
Cloning into 'elts-8.7-release'...
fatal: remote error: 
  Repository not found.
```

**Review by andreasfernandez — CHANGES_REQUESTED** [review]

File: .env (hunk 1 above)
Comment (andreasfernandez) [reply .env:69]: Interestingly, this works with public repositories. It seems with `git://` there's no access check.

Comment (alexander-nitsche) [reply .env:69]: Thanks for testing this.

**Comment by alexander-nitsche** [comment]
Stop. `git@github.com:` should be kept and `git://github.com/` be dropped instead. See https://github.com/TYPO3-Documentation/T3DocTeam/issues/163 for further details. Closing this PR.

**Pull request closed without merging**